modules/gke-cluster-standard/variables.tf

```diff
@@ -113,6 +113,21 @@ variable "cluster_autoscaling" {
   }
 }
 
+variable "default_node_pool" {
+  description = "Enable default node-pool."
+  type = object({
+    remove_pool        = optional(bool, true)
+    initial_node_count = optional(number)
+  })
+  default  = {}
+  nullable = false
+
+  validation {
+    condition     = ((var.default_node_pool.remove_pool == true && var.default_node_pool.initial_node_count == null) || (var.default_node_pool.remove_pool == false && var.default_node_pool.initial_node_count != null))
```

Comment:
I think this is the same a this:
```hcl
    condition = (var.default_node_pool.remove_pool != (var.default_node_pool.initial_node_count == null))
```

Reply:
The condition does not seem to work on my test. :confused:

Reply:
But working with:
```
    condition = (var.default_node_pool.remove_pool != (var.default_node_pool.initial_node_count != null))
```

Reply:
Fixed in 56c81aea7a66bcbfcb0e8d66d03c9e36a0600ec3 :slightly_smiling_face: